packages/anon-aadhaar-pcd/src/pcd.ts

```diff
@@ -45,7 +46,7 @@ export async function init(args: PCDInitArgs): Promise<void> {
 export async function prove(args: IdentityPCDArgs): Promise<IdentityPCD> {
   if (!initArgs) {
     throw new Error(
-      'cannot make semaphore signature proof: init has not been called yet'
+      'cannot make Country Identity proof for aadhaar card: init has not been called yet'
```

Comment:
I think we can get rid of the name "country identity" as discussed

'cannot make Anon Aadhaar proof: init has not been called yet'